docs/examples.md

```diff
@@ -420,6 +420,65 @@ object PerftConverter extends IOApp.Simple {
 ```
 @:@
 
+## Writing data to a CSV file
+
+If you want to save a list of a case class into a CSV file this helper method may aid you:
+
+@:select(scala-version)
+
+@:choice(scala-3)
+```scala mdoc:reset:silent
+import fs2.io.file.Files
```

Comment:
```suggestion
//> using lib "org.typelevel::toolkit::@VERSION@"

import fs2.io.file.Files
```

Reply:
Shouldn't this be `latest` instead of `@version@`?